node/sc/multi_bridge_test.go

```diff
@@ -121,3 +121,31 @@ func TestSetCounterPartBridge(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, dummy, cBridge)
 }
+
+// TestRegisterDeregisterToken checks the following:
+// - the bridge contract method RegisterToken and DeregisterToken.
+func TestRegisterDeregisterToken(t *testing.T) {
+	info := prepareMultiBridgeTest(t)
+	dummy1 := common.Address{10}
+	dummy2 := common.Address{20}
+
+	opts := &bind.TransactOpts{From: info.acc.From, Signer: info.acc.Signer, GasLimit: gasLimit}
```

Comment:
Maybe you can use `info.add` for `opts` for simulated backend. :)

Reply:
I checked it out, but it did not working.